README.md

```diff
@@ -107,17 +107,38 @@ Follow the guidelines in [CONTRIBUTING.md](https://github.com/CSE-25/quick_start
 
 ### Run the package.
 
-1. Run the following command in the terminal to start the package:
+To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the main quick_start_express directory.
+
+1. **Link the Package in the Main Directory**: In the quick_start_express directory, run:
 
    ```bash
-   npm link # in the quick_start_express directory
+   npm link
    ```
 
+2. **Create a Testing Directory**: Outside the quick_start_express directory, create a testing directory:
+
    ```bash
-   npm link qse # in the target directory
+   mkdir ../qse-test
+   cd ../qse-test
    ```
 
-2. Run any qse commands in the target directory such as `qse init`, `qse clear`, `qse -v` ... etc.
+3. **Link qse in the Testing Directory**: In the testing directory, run:
+
+   ```bash
+   npm link qse
+   ```
+   
+4. **Run qse Commands**: Now, you can execute any qse commands in the testing directory, such as:
+
+   ```bash
+   qse init
+   qse clear
+   qse -v
+   ```
+
+**Note**: Running npm link qse in the main directory may modify package.json. Always use a separate testing directory to avoid this.
```

Comment:
```suggestion
> [!Note] 
> Running npm link qse in the main directory may modify `package.json`. Always use a separate testing directory to avoid this.
```

Refer https://github.com/orgs/community/discussions/16925 to use GitHub `md` styles like these.